ui/tests/acceptance/job-detail-test.js

```diff
@@ -1,361 +1,38 @@
-import { get } from '@ember/object';
 import { click, findAll, currentURL, find, visit } from 'ember-native-dom-helpers';
-import moment from 'moment';
 import { test } from 'qunit';
 import moduleForAcceptance from 'nomad-ui/tests/helpers/module-for-acceptance';
+import moduleForJob from 'nomad-ui/tests/helpers/module-for-job';
 
-const sum = (list, key) => list.reduce((sum, item) => sum + get(item, key), 0);
+moduleForJob('Acceptance | job detail (batch)', () => server.create('job', { type: 'batch' }));
+moduleForJob('Acceptance | job detail (system)', () => server.create('job', { type: 'system' }));
+moduleForJob('Acceptance | job detail (periodic)', () => server.create('job', 'periodic'));
```

Comment:
It works, but I'd prefer something that is a little more qunit-y. I sorta fell back into my mocha comfort zone when writing this and nested related tests in the `moduleForJob` function rather than setting some global context with `moduleForJob` and calling the tests right below it.